tmt/schemas/test.yaml

```diff
@@ -22,7 +22,7 @@ properties:
   component:
     $ref: "/schemas/common#/definitions/one_or_more_strings"
 
-  # https://tmt.readthedocs.io/en/stable/spec/tests.html#contact
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#contact
   contact:
     $ref: "/schemas/common#/definitions/one_or_more_strings"
```

Comment:
We should point to the core defintion here as well.

Reply:
Updated.